canonical-websites/snapcraft.io
# Fix issue with canceling pending release

Fixes #1409 

Makes sure canceling pending revisions is properly propagated to revision history component.
Moves `pendingReleases` data from component state into Redux as part of #1384

### QA
- ./run or demo
- go to Releases page of any snap
- promote some revisions or a channel
- click on a cell with pending release (yellow background)
- pending release should appear on top of history of given channel-architecture
- click X in the cell to cancel this release
- promoted release should disappear from the cell and from the top of history
- make sure other functions around releasing work as expected
